docs/specifications/tests/Basic-TP/basic02.md

```diff
@@ -1,45 +1,65 @@
-## BASIC02: The domain must have at least one working name server
+# BASIC02: The domain must have at least one working name server
 
-### Test case identifier
+## Test case identifier
 **BASIC02** The domain must have at least one working name server
 
-### Objective
+## Objective
 
 In order for the domain to work, it must have at least one name server that
 can answer queries about the domain. 
 
-### Inputs
-
-The label of the domain name to be tested.
-
-### Ordered description of steps to be taken to execute the test case
-
-1. Retrieve the IP addresses from the parent delegation using [Method 4](
-   ../Methods.md#method-4-obtain-glue-address-records-from-parent). For name
-   server that are out-of-bailiwick, do separate recursive queries to retrieve
-   the IP addresses of those names.
-2. An NS query for the domain name should be sent to each address.
-3. If there are no valid DNS packets containing the NS answer from any
-   of the name servers, this test case fails.
-
-### Outcome(s)
-
-If at least one recorded response is a DNS response holding one or more NS
-records for the child domain, this test succeeds.
-
-### Special procedural requirements
+## Inputs
+
+The label of the domain name to be tested (the child zone).
+
+## Ordered description of steps to be taken to execute the test case
+
+1. Retrieve the NS records for the child zone (the delegation) using
+   [Method 2]. If the test is an undelegated test, use the name servers
+   from input data. 
```

Comment:
Here you're introducing a reference to input data. Please add an item for this in the Inputs section. Give it a name and refer to that here.